src/public/modules/forms/base/components/feedback-form.client.component.js

```diff
@@ -31,26 +31,9 @@ function feedbackController(FormFeedback, Toastr) {
         isPreview: vm.isPreview,
       }
 
-      FormFeedback.postFeedback(
-        {
-          formId: vm.formId,
-        },
-        feedback,
```

Comment:
Note that the `feedback` object contains whether the feedback is a preview feedback or not:
```ts
const feedback = {
  rating: vm.rating,
  comment: vm.comment,
  isPreview: vm.isPreview, // <-- here
}
```

If we do the 2nd suggestion I outlined above, we can set `vm.isSubmitted` and `vm.isLoading` to the success state immediately when feedback is a preview feedback, and only call the `FormFeedback.postFeedback` if not a preview.